apps/src/studio/controls.html.ejs

```diff
@@ -1,16 +1,16 @@
 <% var msg = require('../locale') %>
 
 <div id="soft-buttons" class="soft-buttons-none">
-  <button id="leftButton" class="arrow">
+  <button id="leftButton" disabled=true class="arrow">
```

Comment:
Will double check. I had thought about it and decided that it shouldn't matter, but always worth checking...